### ISSUE TITLE
Error using T.enum (v1.1.3)

### ISSUE THREAD

**Issue by elmato** [guyskk/validr#44]
```shell
python3 --version
Python 3.6.9

pip3 --version
pip 20.0.2 from /home/yumi/.local/lib/python3.6/site-packages/pip (python 3.6)

pip3 install validr
Processing => validr-1.1.3-cp36-cp36m-linux_x86_64.whl

python3
>>> from validr import T
>>> T.enum([1,2,3])
....
  File "/home/yumi/.local/lib/python3.6/site-packages/validr/schema.py", line 424, in _check_items
    raise SchemaError('items must be bool, int, float or str')
validr._exception_c.SchemaError: items must be bool, int, float or str

>>> T.enum(1,2,3)
...
File "/home/matu/.local/lib/python3.6/site-packages/validr/schema.py", line 362, in __call__
    raise SchemaError("can't call with more than one positional argument")
validr._exception_c.SchemaError: can't call with more than one positional argument
```

**Comment by guyskk** [comment]
T.enum is new feature in v1.2.0 and not released yet #37 . 
 
You can create custom validator as below example for v1.1.3:
https://github.com/anyant/rssant/blob/master/rssant_common/validator.py#L224

**Comment by guyskk** [comment]
v1.2.0 just released, you can install it: `pip install validr==1.2.0` and try it out.
You can also check [changelog](https://github.com/guyskk/validr/blob/master/CHANGELOG.md) and [v1.2.0 check list](https://github.com/guyskk/validr/issues/37).
